### DISCUSSION TITLE
Is it possible to add the spoiler tag?

### DISCUSSION THREAD

**Discussion by YeYuMo** [umputun/remark42#1208] [Ideas]
Hello.

I have using disqus and that tag is used by many users. If I import, how are those comments? I imagine `<spoiler>anything</spoiler> `remains this way as it is not supported. Is it possible to add it for future comments?

Thanks!

**Comment by umputun** [comment]
the regular `<details>` tag already supported, i.e. 

```
<details>
  <summary>spoiler alert</summary>
  The `<details>` tag specifies additional details that the user can open and close on demand.  
  The `<details>` tag is often used to create an interactive widget that the user can open and close. 
  By default, the widget is closed.  When open, it expands, and displays the content within.  
  Any sort of content can be put inside the `<details>` tag.  
</details>
```
<img width="669" alt="g8m5r-202112-26130317-9acib" src="https://user-images.githubusercontent.com/535880/147417638-f576f5eb-0acf-4972-8843-986b3fd98e17.png">

<img width="729" alt="omxsk-202112-26130352-grn27" src="https://user-images.githubusercontent.com/535880/147417648-28d13d5d-b601-4393-8ebb-b094533b87c2.png">

I'm not sure what the exact format for `<details>` in disqus, but I guess you should be able to search/replace  in the imported xml.
